Assets/Scripts/PlayerMovement.cs

```diff
@@ -1,18 +1,54 @@
 using System.Collections;
 using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Diagnostics;
+using System.Security.Cryptography;
+using System.Threading;
 using UnityEngine;
 
 public class PlayerMovement : MonoBehaviour
 {
+
+    public float speed;
+    private Rigidbody2D myRigidbody;
```

Comment:
Maybe rename this to something like playerRigidbody?

Reply:
fixed